src/content/docs/new-relic-solutions/observability-maturity/customer-experience/customer-experience-quality-foundation-guide.mdx

```diff
@@ -77,7 +77,7 @@ This KPI measures whether or not your application or its pages can be accessed b
     id="core-web-lcp-kpi"
     title="Largest contentful paint (LCP)"
   >
-This KPI is part of [Core Web Vitals](https://web.dev/vitals/). Largest Contentful Paint (LCP) measures the time it takes to load the largest image after a user has navigated to a new page.
+This KPI is part of [Core Web Vitals](https://web.dev/vitals/). Largest contentful paint (LCP) measures the time it takes to load the largest image after a user has navigated to a new page.
```

Comment:
calling out again, but assuming it's stylized this way because it's referring to a chrome thing rather than the concept of cwv